requirements.txt

```diff
@@ -26,9 +26,9 @@ requests==2.32.3
 tenacity==9.0.0
 testimony==2.4.0
 wait-for==1.2.0
-wrapanapi
+wrapanapi==3.6.6
```

Comment:
@evgeni If I understand how dependabot works, this change won't help us. Dependabot will still try to bump the version

Reply:
This is the latest version of wrapanapi. Dependabot can bump it as it pleases.

The intent here is to force it (and get a clear error "could not install version <X>" if it can't), instead of picking an ancient version that's broken even more than the current one (2.7.0 in todays case)

Reply:
You probably need to add wrapanapi to `.github/dependabot.yml` to avoid running into the issue again.
Like https://github.com/SatelliteQE/robottelo/blob/master/.github/dependabot.yml#L20

Reply:
Why? I want it to be bumped. I just don't want to allow "any" version for pip/uv/whatever to pick from.

Reply:
ah okay